### DISCUSSION TITLE
How to debug saleor?

### DISCUSSION THREAD

**Discussion by gorjan-mishevski** [saleor/saleor#7349] [Q&A]
I am trying to add a new plugin. It would be much easier if debugging was enabled.
Did someone had any luck with this?
More specifically with InteliJ, how can you link a remote debugger to the docker container on saleor?

**Comment by karolkielecki** [comment]
@gorjan-mishevski Hi! Theoretically, it's possible to do that--to link a debugger. But we haven't done it before. Please read the debugger's documentation. Maybe you can find some info here: https://medium.com/swlh/remote-debugging-a-java-application-running-in-docker-container-with-intellij-idea-efe54cd77f02

**Reply by gorjan-mishevski** [reply]
Hi @karolkielecki, will take a look there, though not sure it would work as it's for attaching on the JVM. I was curious how you guys at Mirumee work on the Saleor core. With a debugger life get's a lot easier, especially for people that have very little understanding of the Python ecosystem (me!)
